src/daemon/collector/content/collect.js

```diff
@@ -241,6 +241,10 @@ function filterFrontAds(lst) {
       && isScrolledIntoView(lst[i], layoutStyle)) {
         newLst.push(lst[i]);
     }
+    if (sponsoredText.includes(lst[i].getAttribute('aria-label'))
```

Comment:
Is this just checking for an `aria-label`? Is that distinguishing enough?

Reply:
for now aria-label of the ad contains the text we can check